core/egov-localization/src/main/java/org/egov/domain/service/MessageService.java

```diff
@@ -104,13 +103,22 @@ public void bustCache() {
         return messages;
     }
     
-    public boolean createMessage(CreateMessagesRequest createMessagesRequest){
-    	messageRepository.deleteMessages(createMessagesRequest);
-   		boolean createStatus = messageRepository.createMessage(createMessagesRequest);
+    public boolean createMessage(String locale, String tenantId, List<Message> messageList){
```

Comment:
Tenant and Locale are independent of the Messages in the Message List. 
To sustain that difference, I've not put those two fields inside the Message Object. 
I needed to have that clear indication that the messages in the message list are for a locale and tenant ID. Thus it is outside the Message Object